mappings/net/minecraft/util/math/random/RandomSequences.mapping

```diff
@@ -0,0 +1,16 @@
+CLASS net/minecraft/class_8565 net/minecraft/util/math/random/RandomSequences
```

Comment:
Since this is a persistent state, should we call it a `RandomSequencesState`?

Reply:
Hmm, not sure. `RaidManager` is also a persistent state without the `State` suffix, but all other subclasses have that suffix.

I don't think it's entirely necessary, but I'm not against it either. Anyone else got input?

Reply:
I do not believe the State prefix should be necessary. However, it should not be a plural; that's usually reserved for utility classes.

Reply:
This class stores and creates instances of `RandomSequence`... felt like a plural was appropriate. If you have a better name, you can suggest it?

Reply:
@Shnupbups State prefix should be used where appropriate- i.e. here.